tensorflow_probability/python/stats/quantiles.py

```diff
@@ -406,7 +406,7 @@ def percentile(x,
                q,
                axis=None,
                interpolation=None,
-               keep_dims=False,
+               keepdims=False,
```

Comment:
I think we will need to keep the existing arg around as deprecated for 3 months.

Can you do something like this:
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/distributions/mvn_diag.py#L138

Perhaps default it to `keep_dims=None`, and then on the first line, `keepdims = keepdims if keep_dims is None else keep_dims`

You can move it to the last position right before name, and add back the docstring but note it's deprecated in the docstring. Date should be +3m from now.

Reply:
Also, this would only apply to public methods. Anything internal you can just change over.